### PR TITLE
updated cta to feature events + secondary link to portal

### PR DESCRIPTION
**Pull request by trulyronak**
# Description

In order to drive more traffic towards the events page, this PR updates the primary call to action (CTA) to point to the events page. It additionally adds another link to the member login page as a secondary link.

Currently still in draft because the member login link looks bad on mobile.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Logistical Change (something about code process / a readme change / documentation update)
- [ ] CI Change (relating to deployment / continuous integration)
- [X] Navigation Change

# How Has This Been Tested?
- [ ] Tested on Localhost
- [ ] Tested on Deployed Preview

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code as needed, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings (if they do, please explain why)

### CODE REVIEW

**Comment by farisashai** [comment]
- finished cta button designs (removed button component that was only used a single time in the entire repo)
- changed css import colors to @use syntax to prepare for sass module transition

File: src/sections/home/Home.Hero.tsx [hunk 1]
```diff
@@ -34,11 +34,17 @@ const HomeHero: React.FC = () => {
...
-                link="https://members.acmucsd.com"
-                type="primary"
-              />
+              <Link href="/events">
+                <a className="hero-btn btn-outlined">Find Events &gt;</a>
+              </Link>
+              <a
+                className="hero-btn"
+                href="https://members.acmucsd.com"
+                target={"_blank"}
```
Comment (duanwilliam) [inline src/sections/home/Home.Hero.tsx:43]: nit: remove curly braces because it's unnecessary

**Review by duanwilliam — APPROVED** [review]
one minor style nit but otherwise LGTM! 🚀

**Review by farisashai — APPROVED** [review]
Shirley approves this message

**Pull request merged**